content/docs/2.0/scalers/metrics-api.md

```diff
@@ -0,0 +1,95 @@
++++
```

Comment:
Would be good to follow same structure from other scalers such as https://github.com/kedacore/keda-docs/blob/master/content/docs/2.0/scalers/redis-streams.md.

Let's talk about the Trigger Specification first, add a section on the API they need and what needs to comply with and end with an example of how to configure it.